### DISCUSSION TITLE
Emmorph részletes elemzés

### DISCUSSION THREAD

**Discussion by kocsrobert** [huspacy/huspacy#72] [General]
**Kedves Fejlesztők.**

Segítségre lenne szükségem. Telepítettem a huspacy modult és jelenleg python-al használom windows-os gépen. Minden modult beépítettem amit szerettem volna, de sajnos a  morfológiai elemzés nem elég részletes. 

**Az e-magyar oldalon a morfológia így néz ki:**
be[/Prev]+gépel[/V]+em[Prs.Def.1Sg]a[/Det|Pro|(Post)]a[/Det|art.Def]a[/N|Ltr]+[Nom]a[/N|Pro|(Post)]+[Nom]hely[/N]+es[_Adjz:s/Adj]+[Nom]hely[/N]+es[_Nz:s/N]+[Nom]helyes[/Adj]+[Nom]kód[/N]+ot[Acc].[OTHER]

**Sajnos a huspacy sajnos nem ad ilyen részletes elemzést. Az outputja ilyen:** MorphDefinite=Def|Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin|Voice=ActDefinite=Def|PronType=ArtCase=Nom|Degree=Pos|Number=SingCase=Acc|Number=Sing

Van rá mód, hogy a huspacyval olyan részletes morfológiai elemzést végezzek mint az e-magyar szoftverrel? Sajnos nincs linux szerver a gépen és emiatt nem szeretnék virtuális operációs rendszert futtatni, mert nagyon bonyolulttá tenné az elemzést. 

Előre is köszönöm a segítséget.

**Comment by oroszgy** [comment]
Egy megoldas lehet az emmorph integracioja, aminek elso lepesirol itt talalsz leirast: https://huspacy.github.io/recipes/emmorph/ Sejtesem szerint viszont Windows alatt ez sem fog mukodni.
